guides/doc-Provisioning_Guide/topics/Cloud-OpenStack.adoc

```diff
@@ -1,22 +1,33 @@
 [[Provisioning_Cloud_Instances_in_Red_Hat_OpenStack_Platform]]
-== Provisioning Cloud Instances in Red Hat OpenStack Platform
+== Provisioning Cloud Instances in {OpenStack}
 
-Red Hat OpenStack Platform provides the foundation to build a private or public Infrastructure-as-a-Service (IaaS) cloud on Red Hat Enterprise Linux. It offers a massively scalable, fault-tolerant platform for the development of cloud-enabled workloads. In {ProjectNameX}, you can interact with Red Hat OpenStack Platforms REST API to create new cloud instances and control their power management states.
+{OpenStack} provides the foundation to build a private or public Infrastructure-as-a-Service (IaaS) cloud. It offers a massively scalable, fault-tolerant platform for the development of cloud-enabled workloads. In {ProjectNameX}, you can interact with {OpenStack} REST API to create cloud instances and control their power management states.
 
 [[Provisioning_Cloud_Instances_in_Red_Hat_OpenStack_Platform-Prerequisites_for_Red_Hat_OpenStack_Platform_Provisioning]]
-=== Prerequisites for Red Hat OpenStack Platform Provisioning:
+=== Prerequisites for {OpenStack} Provisioning:
 
-Requirements for Red Hat OpenStack Platform Provisioning include:
+Requirements for {OpenStack} Provisioning include:
 
-  * Synchronized content repositories for Red Hat Enterprise Linux 7. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red Hat Repositories] in the _Content Management Guide_.
+ifeval::["{Build}" == "foreman"]
+  * The installation media that you require for the operating systems you want to use to provision using Amazon EC2.
+endif::[]
+ifeval::["{Build}" == "satellite"]
+  * Synchronized content repositories for Red{nbsp}Hat Enterprise Linux. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red{nbsp}Hat Repositories] in the _Content Management Guide_.
+endif::[]
   * A {SmartProxyServer} managing a network in your OpenStack environment. For more information, see xref:Configuring_Networking[].
-  * An image added to OpenStack Image Storage (glance) service for image-based provisioning. For more information, see the https://access.redhat.com/documentation/en/red-hat-openstack-platform/9/paged/instances-and-images-guide/[Red Hat OpenStack Platform Instances and Images Guide].
+  * An image added to OpenStack Image Storage (glance) service for image-based provisioning. For more information, see the https://access.redhat.com/documentation/en/red-hat-openstack-platform/9/paged/instances-and-images-guide/[{OpenStack} Instances and Images Guide].
```

Comment:
Need to check if this works actually: `[{OpenStack} Instances and Images Guide]`

Reply:
And it works.

Reply:
:) I tested it out in the preview before raising the PR :D 
Thank you.